chimney/src/main/scala/io/scalaland/chimney/internal/compiletime/derivation/transformer/rules/TransformFromOptionToOptionRuleModule.scala

```diff
@@ -0,0 +1,29 @@
+package io.scalaland.chimney.internal.compiletime.derivation.transformer.rules
+
+import io.scalaland.chimney.internal.compiletime.Definitions
+import io.scalaland.chimney.internal.compiletime.DerivationResult
+import io.scalaland.chimney.internal.compiletime.derivation.transformer.Derivation
+
+trait TransformFromOptionToOptionRuleModule { this: Definitions & Derivation =>
+
+  object TransformFromOptionToOptionRule extends Rule {
+
+    def isApplicableTo[From, To](implicit ctx: TransformerContext[From, To]): Boolean =
+      (Type.Option(Type.Any) <:< Type[From]) && (Type.Option(Type.Any) <:< Type[To])
```

Comment:
Order of subtyping operands seem to be incorrect. Should be `Type[From] <:< Type.Option(Type.Any) && Type[To] <:< Type.Option(Type.Any)`.

WDYT about having some utilities so that we only call `Type[From].isOption`?